tests/data/graph/querybuilder/sample_models/fake_emps_githubusers.py

```diff
@@ -0,0 +1,44 @@
+from dataclasses import dataclass
+
+from cartography.models.core.common import PropertyRef
+from cartography.models.core.nodes import CartographyNodeProperties
+from cartography.models.core.nodes import CartographyNodeSchema
+from cartography.models.core.relationships import CartographyRelProperties
+from cartography.models.core.relationships import CartographyRelSchema
+from cartography.models.core.relationships import LinkDirection
+from cartography.models.core.relationships import make_target_node_matcher
+from cartography.models.core.relationships import OtherRelationships
+from cartography.models.core.relationships import TargetNodeMatcher
+
+
+@dataclass(frozen=True)
+class FakeEmpToGitHubUserRelProperties(CartographyRelProperties):
+    lastupdated: PropertyRef = PropertyRef('lastupdated', set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+class FakeEmpToGitHubUser(CartographyRelSchema):
+    target_node_label: str = 'GitHubUser'
+    target_node_matcher: TargetNodeMatcher = make_target_node_matcher(
+        {'username': PropertyRef('github_username', ignore_case=True)},
```

Comment:
side note: the new changes make it a bit easier to start matching against multiple properties.

Reply:
I guess it makes it more flexible, though I don't want it to grow too much more beyond what it's already at.

We already support matching multiple attrs - see
https://github.com/lyft/cartography/blob/c4310e5b068ff85b9e2054eda115581ea1484883/tests/integration/cartography/graph/test_querybuilder_match_on_multiple_attrs.py#LL19C40-L19C40
and
https://github.com/lyft/cartography/blob/c4310e5b068ff85b9e2054eda115581ea1484883/tests/data/graph/querybuilder/sample_models/multiple_attr_match.py#L26-L31

Reply:
That's right. I somehow forgot.